client/dist/src/components/bars.jsx

```diff
@@ -1,37 +1,93 @@
 import React from 'react';
 import axios from 'axios';
+import Styles from '../css/bars.css';
+
+
+const calculateAverage = (response) => {
+  var total = 0;
+  for (let i = 0; i < response.data.length; i++) {
+    total += response.data[i].prices;
+  }
+  return total / response.data.length;
+}
+
+const findCurrentPrice = (response) => {
+  var currentPrice = 0;
+  for (let i = 0; i < response.data.length; i++) {
+    if (response.data[i].dates === 29) {
+      currentPrice = response.data[i].prices;
+    }
+  }
+  return currentPrice;
+}
+
+const marketIsOpen = () => {
+  var date = new Date();
+  var currentHour = date.getHours();
+  if (currentHour > 15 && currentHour < 6) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 class Bars extends React.Component {
     constructor(props) {
       super(props)
       this.state = {
```

Comment:
I believe that you want to keep state at the root level of your app whenever possible. Something to look into.